webserver/templates/datasets/evaluate.html

```diff
@@ -20,6 +20,12 @@ <h2 class="page-title">Evaluate dataset "{{ dataset['name'] }}"</h2>
         <label class="col-sm-2 control-label">{{ form.filter_type.label.text }}</label>
         <div class="col-sm-4">{{ form.filter_type(class="form-control", required="required") }}</div>
       </div>
+      {% if config['EVAL_LOCATION'] %}
```

Comment:
It might also be better to call this flag `FEATURE_ EVAL_LOCATION` or `FEATURE_REMOTE_EVAL`.